data/reusables/saml/about-linked-identities.md

```diff
@@ -1,3 +1,3 @@
-You can view the single sign-on identity that a member has linked to their {% data variables.product.product_name %} account. When available, the entry will include SCIM data. For more information, see "[About SCIM](/organizations/managing-saml-single-sign-on-for-your-organization/about-scim)."
+You can view the single sign-on identity that a member has linked to their {% data variables.product.product_name %} account. If SAML SSO is configured on organisation, when available, the entry will include SCIM data. For more information, see "[About SCIM](/organizations/managing-saml-single-sign-on-for-your-organization/about-scim)."
```

Comment:
Let's just remove the SCIM part from the reusable and add it directly to "Viewing and managing a member's SAML access to your organization" right after the reference to the reusable.

```suggestion
You can view the single sign-on identity that a member has linked to their {% data variables.product.product_name %} account.
```